.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and Release binary on Push to Master
+on:
+  release:
+    types: [published]
+    branches:
+      - main
+jobs:
+  generate_binary:
+    name: Create binary release
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup elixir
+        id: beam
+        uses: erlef/setup-beam@v1
+        with:
+          otp-version: 25.x # Define the OTP version [required]
+          elixir-version: 1.14.x # Define the elixir version [required]
+          experimental-otp: true
+      - name: Cache Mix
+        uses: actions/cache@v3
+        with:
+          path: deps
+          key: ${{ runner.os }}-mix-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+          restore-keys: |
+            ${{ runner.os }}-mix-
+      - name: Cache Burrito OTP  deps
```

Comment:
So that we don't pose too much of a load on Burrito's CDN when pulling Erlang and OTP deps